src/three/DebugTilesRenderer.js

```diff
@@ -344,6 +345,20 @@ export class DebugTilesRenderer extends TilesRenderer {
 							break;
 
 						}
+						case CUSTOM_COLOR_MODE: {
+
+							if ( this.customDebugColor ) {
+
+								this.customDebugColor( tile, c );
+
+							} else {
+
+								console.error( 'customDebugColor callback not defined' );
```

Comment:
Nit: Lets change this to a warning and prefix it with the class so it's more clear where the error is coming from:

```js
console.error( 'DebugTilesRenderer: customDebugColor callback not defined' );
```

Reply:
Oh yes, that's a lot nicer!